x/sequencer/keeper/hook_listener.go

```diff
@@ -37,9 +37,24 @@ func (hook rollappHook) BeforeUpdateState(ctx sdk.Context, seqAddr string, rolla
 		return types.ErrInvalidSequencerStatus
```

Comment:
isnt this check redundant if we check later if the seq is the proposer?

Reply:
removed